src/applications/disability-benefits/996/pages/informalConference.js

```diff
@@ -90,27 +82,52 @@ const informalConference = {
     },
     informalConferenceTimes: {
       'ui:title': InformalConferenceTimes,
-      'ui:required': formData => formData?.informalConference !== 'no',
-      'ui:errorMessages': {
-        required: errorMessages.informalConferenceTimesMin,
-      },
-      'ui:validations': [checkConferenceTimes],
-      time0800to1000: scheduleTimeUiSchema('time0800to1000'),
-      time1000to1230: scheduleTimeUiSchema('time1000to1230'),
-      time1230to1400: scheduleTimeUiSchema('time1230to1400'),
-      time1400to1630: scheduleTimeUiSchema('time1400to1630'),
       'ui:options': {
         showFieldLabel: true,
         hideIf: formData => formData?.informalConference === 'no',
         expandUnder: 'informalConference',
         forceDivWrapper: true,
-        updateSchema: (formData, schema, uiSchema) => {
-          Object.keys(informalConferenceTimeAllLabels).forEach(time => {
-            const options = uiSchema[time]['ui:options'] || {};
-            // pass informalConference to children
-            options.informalConference = formData?.informalConference;
-          });
-          return schema;
+        updateSchema: formData => {
+          const time1Setting = formData?.informalConferenceTimes?.time1 || '';
+          const time2Setting = formData?.informalConferenceTimes?.time2 || '';
+          const enums = Object.keys(informalConferenceTimeAllLabels);
+          const enumNames = Object.values(informalConferenceTimeAllLabels);
+          return {
+            type: 'object',
+            properties: {
+              time1: {
+                type: 'string',
+                enum: enums.filter(val => val !== time2Setting),
+                enumNames: enumNames.filter(
+                  label =>
+                    label !== informalConferenceTimeAllLabels[time2Setting],
+                ),
+              },
+              time2: {
+                type: 'string',
+                enum: enums.filter(val => val !== time1Setting),
+                enumNames: enumNames.filter(
+                  label =>
+                    label !== informalConferenceTimeAllLabels[time1Setting],
+                ),
+              },
+            },
+          };
+        },
+      },
+      time1: {
+        'ui:title': informalConferenceTimeTitles.first,
+        'ui:required': formData => formData?.informalConference !== 'no',
+        'ui:validations': [checkConferenceTimes],
+        'ui:errorMessages': {
+          required: errorMessages.informalConferenceTimes,
+        },
+      },
+      time2: {
+        'ui:title': informalConferenceTimeTitles.second,
+        'ui:options': {
+          // hideIf: isFirstConferenceTimeEmpty,
```

Comment:
Can this be removed or will it be used at some point down the road??

Reply:
I only wanted that time select to show once the user made a choice, but it never hid... I was going to ask if someone would test it, but I think maybe I'll just remove it since it might be better a11y-wise to always have it showing.

Reply:
Gone :D

Reply:
Ohhh I see. Could you use `expandUnderCondition` to achieve that effect?